epictrack-api/src/api/services/work_status.py

```diff
@@ -22,6 +26,51 @@ class WorkStatusService:  # pylint: disable=too-many-public-methods
     # pylint: disable=too-few-public-methods,
 
     @classmethod
-    def find_all_works_tatus(cls, work_id):
+    def find_all_work_status(cls, work_id):
         """Find all status related to a work"""
         return WorkStatusModel.find_by_params({"work_id": work_id})
+
+    @classmethod
+    def find_work_status_by_id(cls, work_id, status_id):
+        """Find all status related to a work"""
+        results = WorkStatusModel.find_by_params({"work_id": work_id, "id": status_id})
+        return results[0] if results else None
+
+    @classmethod
+    def create_work_status(cls, work_id, work_status: Dict):
+        """Creates a work status."""
+        work_status = WorkStatusModel(
+            **work_status,
+            posted_by=TokenInfo.get_username(),
+            work_id=work_id
+        )
+        work_status.save()
```

Comment:
Does it require to store the result in a variable like work_status = work_status.save() to have the created "id" in the result. I am not sure.

Reply:
i dont think so.. but will check the response to be sure..